src/main/scala/intellij/haskell/external/execution/GhcVersion.scala

```diff
@@ -0,0 +1,14 @@
+package intellij.haskell.external.execution
```

Comment:
Could be moved to HaskellProjectUtil together with the stack call to get GHC version. In this way it can be reused and I like to have the execution and parsing the result together.